Gemfile

```diff
@@ -12,6 +12,11 @@ gem "config"
 gem "bootsnap", require: false
 gem "puma"
 gem "rack-utf8_sanitizer"
+gem "net-smtp"
+gem "net-imap"
+gem "net-pop"
+gem "tzinfo-data"
+gem "webrick"
```

Comment:
What's this for?

Reply:
In Ruby 3 these are no longer bundled gems or standard libraries, so they need to be included in the Gemfile. If I recall, `webrick` and `tzinfo-data` are definitely needed here, though I can check those and the other ones one by one to make sure

Reply:
See "Other notable changes since 2.7" in https://www.ruby-lang.org/en/news/2020/12/25/ruby-3-0-0-released/